apps/meteor/app/livechat/client/stylesheets/livechat.css

```diff
@@ -146,3 +146,11 @@
 .contact-chat-history-search-empty {
 	padding-left: 24px;
 }
+
+.rcx-message-system__block {
+	align-items: flex-start !important;
+}
+
+.rcx-message-system__body {
+	white-space: pre-wrap !important;
+}
```

Comment:
this is not the place where the changes should be placed, please look for the message component

Reply:
Can you check now again?

Reply:
Yes, that's true, I tried it with @rocket.chat/css-in-js, but it's not working with it. 

any CSS file? where you handle Message Component custom styling?

Reply:
take a look at the Fuselage's component!

Reply:
https://rocketchat.github.io/fuselage/fuselage/master/?path=/story/message-messagesystem--default

not find any specific for this ...